apps/recovery-portal/src/main/webapp/errors/error_401.jsp

```diff
@@ -44,7 +44,6 @@
 <body class="login-portal layout authentication-portal-layout">
     <layout:main layoutName="<%= layout %>" layoutFileRelativePath="<%= layoutFileRelativePath %>" data="<%= layoutData %>" >
         <layout:component componentName="ProductHeader" >
-            <!-- product-title -->
```

Comment:
any reason to remove this here?

Reply:
Seems like I've missed this one. Will fix this and have a look about any other missings too